src/integrationTest/java/uk/gov/hmcts/dts/fact/repositories/CourtRepositoryTest.java

```diff
@@ -74,6 +74,15 @@ void shouldFindApplicationUpdatesFromNotInPersonCourt() {
         assertThat(courtApplicationUpdates.get(0).getApplicationUpdate()).isNotNull();
     }
 
+    @Test
+    void shouldFindAdditionalLinkInCourt() {
+        final Optional<Court> result = courtRepository.findBySlug("west-london-family-court");
+        assertThat(result).isPresent();
+        final List<CourtAdditionalLink> courtAdditionalLinks = result.get().getCourtAdditionalLinks();
+        assertThat(courtAdditionalLinks).isNotEmpty();
+        assertThat(courtAdditionalLinks.get(0).getAdditionalLink()).isNotNull();
```

Comment:
```suggestion
        assertThat(courtAdditionalLinks.get(0).getAdditionalLink()).isNotNull();
        
        Should we also add checks for the url and description to make sure they aren't null? Or to more specifically check the value comes back as we expected. Suggestion! As it might be a bit overkill
```

Reply:
The idea of this type of repository test is to test that we can get information back from the table without paying too much attention to the actual value as they can change over time. Proper testing needs to be in place on the front end functional test. Think we can test the url and description are not null too